app/models/patient.rb

```diff
@@ -549,7 +549,8 @@ def send_assessment(force = false)
     # NOTE: We do not close out folks on the non-reporting line list in exposure (therefore monitoring will still be true for them),
     # so we also have to check that someone receiving messages is not past they're monitoring period unless they're  in isolation,
     # continuous exposure, or have active dependents.
-    return unless (monitoring && (!last_date_of_exposure.nil? && last_date_of_exposure >= ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day)) ||
+    start_of_exposure = last_date_of_exposure || created_at
+    return unless (monitoring && start_of_exposure >= ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day) ||
```

Comment:
I guess technically it should be impossible for `created_at` to be `nil`, but thoughts on keeping a nil check anyway here to avoid the error if it ever did occur for some unknown reason?

Reply:
I'd actually like to see that pop up in Sentry (if it happened)